src/mcts/search.cc

```diff
@@ -450,27 +454,30 @@ EdgeAndNode Search::GetBestChildWithTemperature(Node* parent,
   assert(parent->GetChildrenVisits() > 0);
   std::vector<float> cumulative_sums;
   float sum = 0.0;
-  uint32_t max_n = 0;
+  float max_n = 0.0;
+  float offset = kTemperatureVisitOffset;
 
   for (auto edge : parent->Edges()) {
     if (parent == root_node_ && !root_limit.empty() &&
         std::find(root_limit.begin(), root_limit.end(), edge.GetMove()) ==
             root_limit.end()) {
       continue;
     }
-    if(edge.GetN() > max_n) {
-      max_n = edge.GetN();
+    if(edge.GetN() + offset > max_n) {
+      max_n = edge.GetN() + offset;
     }
   }
-  assert(max_n);
+  assert(max_n > 0.0);
```

Comment:
Is it intended to crash if the offset parameter is too low?

Reply:
I've changed the bounds for the offset parameter to make it clear that it shouldn't be set lower than or equal to -1 so this should only ever crash if someone violates the parameter constraints.